detection_rules/devtools.py

```diff
@@ -157,6 +157,17 @@ def build_integration_docs(ctx: click.Context, registry_version: str, pre: str,
     return docs
 
 
+@dev_group.command('test-new-docs')
+def test_new_docs():
```

Comment:
This will be removed. Waiting on https://github.com/elastic/detection-rules/pull/2715 to close